otter/convergence/gathering.py

```diff
@@ -138,6 +141,22 @@ def get_scaling_group_servers(tenant_id, group_id, now,
     yield do_return(servers)
 
 
+def get_all_stacks(stack_tag=None, batch_size=100):
+    query = {}
+
+    if stack_tag is not None:
+        query['tags'] = stack_tag
+
+    return list_stacks_all(query)
+
+
+@do
+def get_scaling_group_stacks(tenant_id, group_id, now,
+                             get_all_stacks=get_all_stacks):
+    stacks = yield get_all_stacks(stack_tag=get_stack_tag_for_group(group_id))
+    yield do_return(stacks)
```

Comment:
Nitpick: Since this basically delegates call to another function, you don't need `do` generator here. This can just be `return get_all_stacks(stack_tag=get_stack_tag_for_group(group_id))`.